tensorboard/plugins/graph/tf_graph_common/parser.ts

```diff
@@ -211,7 +211,7 @@ export function parseStatsPbTxt(input: ArrayBuffer):
  */
 function parsePbtxtFile(
     input: ArrayBuffer,
-    repeatedFields: {[attrPath: string]: boolean}): Promise<Object> {
+    repeatedFields: {[attrPath: string]: boolean}): Promise<any> {
```

Comment:
Does using `tf.graph.proto.NodeDef[]` in place of `any` work? It'd be nice to avoid `any` when working with libraries we can control. Thanks a lot for the fix.

Reply:
Unfortunately, no.  It seems parsePbtxtFile is used in a multiple places in this file with different return types.  It's genuinely a function that returns an arbitrary type (from the code, it looks like it's deserializing data into an arbitrary shape), and at some point we must "downcast" the result.

(See parseGraphPbTxt and parseStatsPbTxt.)

Reply:
We use any in more than a few places, which would be nice to clean up some day. I'm happy merging this for now, so our build continues working.